Geometry/HGCalCommonData/python/cmsExtendedGeometry2023D28AXML_cfi.py

```diff
@@ -0,0 +1,120 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
@bsunanda - this file should be generated from 2023 dictionary

Reply:
I made this using 2023 dictionary in a private way. I discussed with Kevin and he does not want this to be made an official workflow to which I completely agreed. So this cfi is to do some non-official studies. I can rename the file if this sounds like an official name.

Reply:
please, do rename it to avoid confusion.

Reply:
on the second thought, it has to be generated from a dictionary even if it's not result in a different workflow. We are going this way to all geometry scenarios in any case.